swap_meet/vendor.py

```diff
@@ -1,2 +1,67 @@
 class Vendor:
-    pass
+
+    def __init__(self, inventory=None):
+        if inventory is None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
```

Comment:
We can also use a ternary here:

```suggestion
        self.inventory = inventory if inventory is not None else []
```
 More info on ternarys: https://www.geeksforgeeks.org/ternary-operator-in-python/